ql/src/test/results/clientpositive/llap/acid_direct_update_delete_partitions.q.out

```diff
@@ -148,30 +156,50 @@ POSTHOOK: Input: default@test_update_part@c=11
 POSTHOOK: Input: default@test_update_part@c=22
 POSTHOOK: Input: default@test_update_part@c=33
 POSTHOOK: Input: default@test_update_part@c=__HIVE_DEFAULT_PARTITION__
+POSTHOOK: Output: default@test_update_part
+POSTHOOK: Output: default@test_update_part@c=11
 POSTHOOK: Output: default@test_update_part@c=11
 POSTHOOK: Output: default@test_update_part@c=22
+POSTHOOK: Output: default@test_update_part@c=22
 POSTHOOK: Output: default@test_update_part@c=33
 POSTHOOK: Output: default@test_update_part@c=__HIVE_DEFAULT_PARTITION__
+POSTHOOK: Output: default@test_update_part@c=__HIVE_DEFAULT_PARTITION__
+POSTHOOK: Lineage: test_update_part PARTITION(c=11).a SIMPLE []
```

Comment:
Why is this lineage change?

Reply:
With early split update we end up with a multi insert statement: all of the branches inserts to the same table. Hive collects the lineage info to a List not a Set and while traversing all insert branches. This leads to duplicates.